### ISSUE TITLE
article types are not unique (image, video, and article)

### ISSUE THREAD

**Issue by cnvo** [anggadarkprince/infogue#4]
When choosing an article type, such as either an image or video, it should be in a different visual format.

**Comment by anggadarkprince** [comment]
Yes this is that I want to do so, if the type is gallery or images, the inputs just upload multiple image with description each, so does the video type works..
